src/Simplic.Studio.Tenant.UI/OrganizationViewModel.cs

```diff
@@ -0,0 +1,264 @@
+﻿using Simplic.TenantSystem;
+using Simplic.UI.MVC;
+using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+
+namespace Simplic.Studio.TenantSystem.UI
+{
+    /// <summary>
+    /// OrganizationViewModel to handle tenant data.
+    /// </summary>
+    public class OrganizationViewModel : ViewModelBase
+    {
+        private IOrganizationService organizationService = CommonServiceLocator.ServiceLocator.Current.GetInstance<IOrganizationService>();
+        private ObservableCollection<Organization> organizations;
+        public int subOrganizationCount;
+        public IList<Guid> subOrganizations;
+        public Guid? cloudOrganizationId;
+        private Organization model;
+        private bool isCreate;
+
+        /// <summary>
+        /// Default constructor for the view model
+        /// </summary>
+        public OrganizationViewModel()
+        {
+            Organizations = new ObservableCollection<Organization>();
+
+            foreach (var organization in organizationService.GetAll())
+            {
+                Organizations.Add(organization);
+            }
+
+            this.IsCreate = true;
+        }
```

Comment:
I havent seen any use of the Organizations collection in this pr and it is usually a bad idea to load all data object to interact with a single one.
Also this constructor is completly unused